apps/web/components/AppListCard.tsx

```diff
@@ -85,52 +86,67 @@ export default function AppListCard(props: AppListCardProps) {
   }, [highlight, pathname, router, searchParams, shouldHighlight]);
 
   return (
-    <div className={classNames(highlight && "dark:bg-muted bg-yellow-100", className)}>
-      <div className="flex items-center gap-x-3 px-4 py-4 sm:px-6">
-        {logo ? (
-          <img
-            className={classNames(logo.includes("-dark") && "dark:invert", "h-10 w-10")}
-            src={logo}
-            alt={`${title} logo`}
+    <>
+      {invalidCredential && (
+        <div className="fixed left-0 top-0 mb-3 w-[100vw]">
+          <TopBanner
+            text={`Oh no! Looks like permission expired or was revoked. Please reinstall again. `}
+            variant="warning"
+            actions={
+              <a href={`/apps/${slug}`} className="border-b border-b-black">
+                Go to App
+              </a>
```

Comment:
Don't we use a Link tag for the navigation?

Reply:
> Don't we use a Link tag for the navigation?


yes we can.